app/konnect/api/identity-management/identity-integration.md

```diff
@@ -0,0 +1,170 @@
+---
+title: End to End Identity Integration
+content-type: how-to
+---
+
+
+## Create a custom team
+
+Custom Teams serve as a primary way for organizations to provision access of users to different entities in the organization. Custom teams are used to mirror the organizational structure in an organization, any user who is a member of a custom team will inherit all of the roles of this team.
+
+You will need to authenticate with the API, for information about authentication read the [API authentication](/konnect/api/index/#authentication) instructions.
+Create a custom team by sending a `POST` request containing the `name` and `description` of your team in the response body: 
+```
+curl --request POST \
+  --url https://global.api.konghq.tech/v2/teams \
+  --header 'Content-Type: application/json' \
+  --data '{
+  "name": "IDM - Developers",
+  "description": "The Identity Management (IDM) team."}'
+```
+
+You will receive a `200` response code, and a response body containing information about your team: 
+
+```
+{
+"id": "a1d5c35a-3c71-4d95-ae4b-438fa9bd1059",
+"name": "IDM - Developers",
+"description": "The Identity Management (IDM) team.",
+"system_team": false,
+"created_at": "2022-10-25T16:39:27Z",
+"updated_at": "2022-10-25T16:39:27Z"
+}
+```
+
+Save the `id` value, so that you can reference this team throughout the guide.
+
+
+### Assign a role to a custom team
+
+In order for a custom team to be useful, roles must be assigned to the team. All custom teams start with 0 roles and each role must be added to the team in order for members of the team to inherit the roles. Roles define a set of permissions or actions that a user is allowed to perform on a {{site.konnect_short_name}} entity. 
+
+1. Obtain a list of available roles by issuing a `GET` a request:
+
+        curl --request GET \
+        --url https://global.api.konghq.tech/v2/roles
+
+   The response body will contain a list of available roles: 
+
+    ```
+    {
+	"analytics": {
+		"name": "Analytics",
+		"roles": {
+			"analytics_admin": {
+				"name": "Analytics Admin",
+				"description": "Allows users to view, create, modify, and delete all content in Konnect Analytics."
+			},
+			"analytics_viewer": {
+				"name": "Analytics Viewer",
+				"description": "Allows users to view all but not modify any content in Konnect Analytics."
+			}
+		}
+	}
+    ```
+
+2. Assign a role to a team by issuing  a `POST` request:
+    
+    The request should contain the `id` as a parameter in the URL. This request requires a JSON response body that contains `role`, `entity_id`, `entity_type`, and `entity_region`. 
+
+        curl --request POST \
+        --url https://global.api.konghq.tech/v2/teams/TEAM_ID/assigned-roles \
+        --header 'Content-Type: application/json' \
+        --data '{
+        "role": "analytics_viewer",
+        "entity_id": "e67490ce-44dc-4cbd-b65e-b52c746fc26a",
+        "entity_type": "runtime_groups",
+        "entity_region": "eu"
+        }'
```

Comment:
This also looks like it's missing a code block, but it looks like it's rendered correctly in the output, so I'm confused